src/GitHub.App/ViewModels/GitHubPane/PullRequestDetailViewModel.cs

```diff
@@ -328,7 +329,7 @@ public Uri WebUrl
                 LocalRepository = localRepository;
                 RemoteRepositoryOwner = owner;
                 Number = number;
-                WebUrl = LocalRepository.CloneUrl.ToRepositoryUrl().Append("pull/" + number);
+                WebUrl = localRepository.CloneUrl.ToRepositoryUrl(owner);
```

Comment:
Did you mean to remove the "pull/number" part from the URL? The links now open the main repository page rather than the PR page.

Reply:
Oh crap, no I didn't! When I tested, I must have only checked it was the correct repo. 😊